### PR TITLE
Module 3: React Capstone - Metrics Webapp

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 3 (Week 5): React Capstone - Metrics Webapp 
- Used the open weather air pollution API for fetching data
- Added the homepage where users can filter by cities
- Added the details page for display air pollution data of specific state
-  Used redux store to manage the details coming from the api
-  Added tests for the various components using Jest and react testing library
- [video link](https://www.loom.com/share/9ba2dedb65f24ed08feda5e9b73bc1f1) project description

### CODE REVIEW

File: src/components/List.module.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  padding: 1em;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+}
+
+ul > li > div > p:nth-of-type(even) {
+  background: linear-gradient(30deg, rgb(67, 81, 111) 85%, rgb(71, 75, 118) 98%);
+}
+
```
Comment (Owusu-Desmond) [inline src/components/List.module.css:25]: - [ ] Please from the project requirement you should make sure that the layout: composition and space between elements matched the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))[design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) But currently you have white spaces in between this list in your project :point_down: Kindly make sure that there is no margin between them as it is in the design guidelines 
  
  ### Your work
  image
  
  ### Expect to have no margins between the list like this :point_down: 
  
  image

File: src/components/List.module.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.subHeading {
+  padding: 1em 0.5em;
+  text-align: right;
+  text-transform: uppercase;
+}
+
+li > div {
+  display: flex;
```
Comment (Owusu-Desmond) [inline src/components/List.module.css:8]: - [ ] You did a great job implementing the design guidelines  :+1:  but please you are missing the voice icon in the nav bar and you have no background color for the section after the nav bar  (the color does **not** need to be the same color as in the design guidelines) so kindly make sure you fix this to make your design looks exactly like the given design. :100: 
  
  ### Your design
  Screenshot from 2023-05-24 00-13-41
  
  ### Given design
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @ikennarichard , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Your test passes :+1: ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,201 @@
...
+- Homepage
+- Details page for displaying state specific data
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](https://rich-air.netlify.app/)
```
Comment (tufoinnkuo10) [inline README.md:78]: - [optional] kindly separate the live demo and video demo, you can put them on two separate lines to avoid confusion. nice work

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ikennarichard 

Your project is complete! Great job making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Your test passes 👍 ✅
📌 Good commit messages ✅
📌 Professional README file ✅

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**